content/api/arduino.md

```diff
@@ -113,7 +170,7 @@ The values are the voltages read on the pins, between 0 and 5.
 Pins `A4` and `A5` are reserved and cannot be used.
 {{% /notice %}}
 
-## Ultrasound Sensors
+## Ultrasound Sensors (Unavailable in Simulator)
```

Comment:
I think we're calling the distance sensors ultrasound sensors in the simulator (they have an ultrasound sensor model, and behave like an ultrasound sensor, except they don't have a minimum distance).

Reply:
Correct. The API is also slightly different.